google-cloud-tools-plugin/google-account/src/com/google/cloud/tools/intellij/login/IntegratedGoogleLoginService.java

```diff
@@ -18,17 +18,15 @@
 
 import com.google.api.client.auth.oauth2.Credential;
 import com.google.api.client.http.HttpRequestFactory;
-
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
 import java.util.Map;
 import java.util.Optional;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Provides Google user authentication services.
  */
-public interface GoogleLoginService {
+public interface IntegratedGoogleLoginService {
```

Comment:
Will we have plans to name these back to remove the redundant `Integrated` prefix in the future?

Reply:
no current plans but open to it once our users are sufficiently migrated over.